markets/perps-market/test/integration/Market/Size.test.ts

```diff
@@ -11,6 +11,7 @@ describe('Market - size test', () => {
         name: 'Ether',
         token: 'snxETH',
         price: bn(2000),
+        maxMarketValue: bn(9999999),
```

Comment:
instead of needing to pass it in everywhere, in the `bootstrapPerpsMarket`, have a default value it can set it to when its not present 👍🏽